assets/i18n/en.json

```diff
@@ -380,6 +377,10 @@
     "I18N_LIBRARY_SUB_HEADER": "Browse the full set of lessons created by the community",
     "I18N_LIBRARY_VIEWS_TOOLTIP": "Views",
     "I18N_LIBRARY_VIEW_ALL": "View all",
+    "I18N_LICENSE_PAGE_LICENSE_HEADING": "License",
+    "I18N_LICENSE_PAGE_PARAGRAPH_1": "All content in Oppia's explorations is licensed under <a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode\" class=\"inline-links\">CC-BY-SA 4.0</a>.",
+    "I18N_LICENSE_PAGE_PARAGRAPH_2": "The software powering Oppia is open source, and its <a href=\"https://github.com/oppia/oppia/\" class=\"inline-links\">code</a> is released under an <a href=\"https://www.apache.org/licenses/\" class=\"inline-links\">Apache 2.0</a> license.",
+    "I18N_LICENSE_TERMS_HEADING": "License Terms",
```

Comment:
@sean Can you check those? For example, I think we might want to also include other stuff than exploration in these texts but not sure.

Reply:
Thanks for flagging @vojtechjelinek. Yes, let's change it -- let's change "explorations" to "lessons".  I think that suffices.

Reply:
done